### PR TITLE
Show how to retrieve workflow in a class

### PR DESCRIPTION
**Pull request by amalricBzh**
Show how to retrieve workflow in a class or a controller using dependency injection.

### CODE REVIEW

File: workflow.rst [hunk 1]
```diff
@@ -215,6 +215,37 @@ what actions are allowed on a blog post::
...
+To access workflow inside a class, use dependency injection and inject the registry in the constructor. For a controller, you can directly inject it in a method::
+
+    use Symfony\Component\Workflow\Registry;
+    
+    class MyClass
+    {
+    
+        private $worflowRegistry;
+    
+        public function __construct()
```
Comment (noniagriconomie) [inline workflow.rst]: you miss to inject `$worflowRegistry` in in the constructor

File: workflow.rst [hunk 2]
```diff
@@ -215,6 +215,37 @@ what actions are allowed on a blog post::
 
     // See all the available transitions for the post in the current state
     $transitions = $workflow->getEnabledTransitions($post);
+    
+In a class or a controller
+--------------------------
+
+To access workflow inside a class, use dependency injection and inject the registry in the constructor. For a controller, you can directly inject it in a method::
```
Comment (noniagriconomie) [inline workflow.rst]: IMO you should go to a new line after the `constructor.`

Comment (amalricBzh) [reply workflow.rst]: Oups, fixed, thank you.

**Comment by wouterj** [comment]
Thanks @amalricBzh! I think this definitely is usefull information to have in the docs. I've slightly changed the title so it's clear what we're talking about here and merged everything in all versions.

**Pull request closed without merging**